jsonapi-utils.gemspec

```diff
@@ -28,4 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec-rails', '~> 3.1'
   spec.add_development_dependency 'factory_girl', '~> 4.5'
   spec.add_development_dependency 'smart_rspec', '~> 0.1.4'
```

Comment:
`smart_rspec` has some dependency problems when we use Rails 5. It's quick fix though, I think you just need to change the active support version in the `smart_rspec` gem file